assets/js/translator.js

```diff
@@ -147,6 +147,9 @@ if(modeEnabled('translation')) {
             if($('div#translateText').is(':visible')) {
                 translateText();
             }
+            else if($('div#docTranslation').is(':visible')) {
+                translateDoc();
+            }
```

Comment:
Um, I did a `git fetch origin, git rebase origin/master`. I am not sure why it still exists.